lib/fhc.js

```diff
@@ -152,7 +152,7 @@ fhc.applyCommandFunction = function (processArgv, cb) {
   // Only new-style commands can be validated
   if (cmdFn.demand) {
     try {
-      argUtils.validate(yargs, cmdFn);
+      argv = argUtils.validate(yargs, cmdFn);
```

Comment:
@wei-lee Could you check the following commands with this PR? Are these commands working?

* bin/fhc.js app read --p=projectID --a=projectID
* bin/fhc.js app read -p=projectID -a=appID
* bin/fhc.js app read p=projectID a=appID
* bin/fhc.js app read projectID appID

Look that is the requirement into https://issues.jboss.org/browse/RHMAP-7260 which originated the Pr #307 . Please, if this alias are working now feel free to close the #307 in favor of this one.